draft-irtf-cfrg-opaque.md

```diff
@@ -318,9 +318,9 @@ pre-computation attacks (as defined in {{JKX18}}). OPAQUE provides forward
 secrecy with respect to password leakage while also hiding the password from
 the server, even during password registration. OPAQUE allows applications
 to increase the difficulty of offline dictionary attacks via iterated
-hashing or other key stretching schemes. OPAQUE is also extensible, allowing
+hashing or other key-stretching schemes. OPAQUE is also extensible, allowing
 clients to safely store and retrieve arbitrary application data on servers
-using only their password.
+using only their passwords.
```

Comment:
```suggestion
using only their password.
```

Reply:
I'm not native in English so it's very probable that might suggestion comes from another language.

But does putting the singular to password not mean something like "the one password that all the clients have" instead of "the password of each client"?

Reply:
I think it is best left signular, agree with Chris's suggestion here